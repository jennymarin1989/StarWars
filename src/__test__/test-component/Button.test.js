import React from 'react';
import { shallow } from 'enzyme';

describe ('Button', () => {

  describe ('renders components', () => {
    it('renders components correctly', () => {
      let app = shallow(<Button />);
     expect(app).toMatchSnapshot(); 
    });
  })
})