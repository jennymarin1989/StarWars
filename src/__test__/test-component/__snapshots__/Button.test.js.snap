// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders components renders components correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Button />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <button
        id="homepage-button"
>
        <img
                alt="starwars"
                src={undefined}
                width="80px"
        />
</button>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="starwars"
          src={undefined}
          width="80px"
/>,
        "id": "homepage-button",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "starwars",
          "src": undefined,
          "width": "80px",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "button",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <button
          id="homepage-button"
>
          <img
                    alt="starwars"
                    src={undefined}
                    width="80px"
          />
</button>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="starwars"
            src={undefined}
            width="80px"
/>,
          "id": "homepage-button",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "starwars",
            "src": undefined,
            "width": "80px",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "button",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
